kraken-build/pyproject.toml

```diff
@@ -1,75 +1,59 @@
 [build-system]
-build-backend = "poetry.core.masonry.api"
-requires = ["poetry-core"]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
```

Comment:
Beware, hatchling does not always include not *.py type. It might be worth to check that `gitignore-io-tokens.json.gz` is still included in the built wheels.

Aside tip: if you want to be cutting-edge UV build backend is now stable: https://docs.astral.sh/uv/concepts/build-backend/